connectors/source.py

```diff
@@ -236,6 +237,22 @@ def tweak_bulk_options(self, options):
         """
         pass
 
+    def _ssl_context(self, certificate):
+        """Convert string to pem format and create a SSL context
+
+        Args:
+            certificate (str): certificate in string format
+
+        Returns:
+            ssl_context: SSL context with certificate
+        """
+        certificate = certificate.replace(" ", "\n")
+        certificate = " ".join(certificate.split("\n", 1))
+        certificate = " ".join(certificate.rsplit("\n", 1))
+        ctx = ssl.create_default_context()
+        ctx.load_verify_locations(cadata=certificate)
+        return ctx
```

Comment:
This function should go to `utilities` module.

Reply:
Alright, will move it to utils module.